Cargo.toml

```diff
@@ -86,7 +86,3 @@ opt-level = "s"
 # ! This will cause the FFI to stop working because UniFFI stores the Rust <-> cdylib mapping
 # ! in the `.strtab` section of the executable. Stripping this causes everything to stop functioning.
 strip = false
-# panic = "abort"
-
-[profile.dev.package.backtrace]
-opt-level = 3
```

Comment:
Why change this in this PR?

Reply:
Ugh, that was automatically done by cargo, not sure why it did it, and I didn't notice. Will remove.
Could be that cargo detected this is a stale entry or something, but in any case that's for a different PR.